src/components/CompendiaExplore.js

```diff
@@ -17,17 +17,25 @@ const ListItem = ({ href, text }) => (
   </Box>
 )
 
-export const CompendiaExplore = () => {
+export const CompendiaExplore = ({ type }) => {
   const { setResponsive } = useResponsive()
 
+  const isNormalized = type === 'normalized'
+  const linkText = `Learn how the normalized ${
+    isNormalized ? 'normalized' : 'RNA-seq'
+  } compendia are created`
```

Comment:
```suggestion
  const isNormalized = type === 'normalized'
  const learnHowLinkText = isNormalized
    ? 'Learn how the normalized compendia are created'
    : 'Learn how the RNA-seq compendia are created'
```

I think being a bit more verbose here is worth the improvement to readability. I would also update the variables to be a bit more specific.